Resources/Prototypes/_NF/Events/nf_bluespace_grids_events.yml

```
@@ -94,7 +94,7 @@
     weight: 5
     duration: 590
     maxDuration: 780
-    reoccurrenceDelay: 480 # 8 hours
+    reoccurrenceDelay: 240 # 4 hours
```

```suggestion
    reoccurrenceDelay: 360 # 4 hours
```
Once in a six hour shift.